### DISCUSSION TITLE
FMT-model能进行固定翼模型在环仿真和开环仿真吗？

### DISCUSSION THREAD

**Discussion by wuhuan666** [Firmament-Autopilot/FMT-Firmware#53] [Q&A] [answered]
如果行的话，毕业设计准备用大佬写的飞控软件。
这个飞控不像pix4，搅合的理不清，底层和算法分离，赞

**Comment by JcZou** [comment, marked as answer]
可以做开环仿真，但是暂时还不能进行模型在环仿真，因为还缺少固定翼的Plant算法模型（目前正在开发中）。当然你也可以自己实现一个固定翼的Plant

**Comment by JcZou** [comment]
可以做开环仿真，但是暂时还不能进行模型在环仿真，因为还缺少固定翼的Plant算法模型（目前正在开发中）。当然你也可以自己实现一个固定翼的Plant
